cli/tests/unit/command_test.ts

```diff
@@ -424,6 +424,26 @@ Deno.test({ permissions: { run: true } }, function commandSyncNotFound() {
   );
 });
 
+Deno.test({ permissions: { run: true } }, function cwdNotFound() {
+  assertThrows(
+    () => new Deno.Command("ls", {
```

Comment:
The `ls` command won't exist on windows. Might be better to use `curl`.

Reply:
How about using `Demo.execPath()` I've noticed most of the tests are using just that

Reply:
Sure.